modules/core/src/main/java/org/apache/ignite/internal/util/worker/IgniteWorker.java

```diff
@@ -109,16 +107,14 @@ public final void run() {
 
             body();
         } catch (InterruptedException e) {
-            if (log.isDebugEnabled()) {
-                log.debug("Caught interrupted exception: " + e);
-            }
+            log.debug("Caught interrupted exception", e);
 
             Thread.currentThread().interrupt();
         } catch (Throwable e) {
             // Catch everything to make sure that it gets logged properly and
             // not to kill any threads from the underlying thread pool.
 
-            log.error("Runtime error caught during ignite runnable execution: " + this, e);
+            log.warn("Runtime error caught during ignite runnable execution [worker={}]", this, e);
```

Comment:
seems need to exchange 'this' and 'e' parameters

Reply:
good catch!